shared/lib/components.js

```diff
@@ -155,8 +155,12 @@ function render(componentName, options) {
   const macroName = componentNameToMacroName(componentName)
   const macroPath = `govuk/components/${componentName}/macro.njk`
 
-  // Return built fixture or render
-  return options?.fixture?.html ?? renderMacro(macroName, macroPath, options)
+  // Use built fixtures (if they exist) on CI to optimise for speed
+  if (process.env.CI === 'true' && options?.fixture?.html) {
```

Comment:
Sorry to realise this a bit late, but we'll want the HEROKU check here too

We optionally pass through the fixture from **app.mjs** with our new check in:

https://github.com/alphagov/govuk-frontend/blob/4f1d252f2f8dbc305fab6910d70f4ee120a0b7b2/packages/govuk-frontend-review/src/app.mjs#L223

But then chuck it away again here on line 159 when running in Heroku

Reply:
E.g. To prevent re-rendering on Heroku page reloads we'd need to add `HEROKU_APP`

```suggestion
  if (options?.fixture?.html && (process.env.CI || process.env.HEROKU_APP)) {
```

Reply:
I believe I've addressed this in https://github.com/alphagov/govuk-frontend/commit/e3651dc8ca9d2163a4eca49df8e3323373390270 – happy with that?

Reply:
Amazing, thank you